FakeNFT/Scenes/StatisticsPage/StatisticsPageViewController.swift

```diff
@@ -0,0 +1,133 @@
+import UIKit
+import ProgressHUD
+
+final class StatisticsPageViewController: UIViewController {
+    private var viewModel: StatisticsPageViewModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .asset(.white)
+        view.addSubview(tableView)
+
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor)
+        ])
+
+        let navigationController = UINavigationController(rootViewController: StatisticsPageViewController())
+
+        navigationController.view.translatesAutoresizingMaskIntoConstraints = false
+        let backButton = UIBarButtonItem()
+        backButton.title = ""
+        navigationItem.backBarButtonItem = backButton
+        navigationItem.rightBarButtonItem = menuButton
+        let model = StatisticsPageModel()
+        viewModel = StatisticsPageViewModel(model: model)
+        viewModel.onChange = updateTable
```

Comment:
Передавая ссылку на функцию ты делаешь вторую сильную связь с объектом(первая var viewModel:). Такие ошибки в каждом модуле, надо исправить
```suggestion
        viewModel.onChange = { [weak self]
            self?updateTable
        }
```